.ci/pytorch/common_utils.sh

```diff
@@ -158,6 +158,27 @@ function install_torchvision() {
   fi
 }
 
+function install_torchrec_and_fbgemm() {
+  local torchrec_commit
+  local fbgemm_commit
+  torchrec_commit=$(get_pinned_commit torchrec)
+  fbgemm_commit=$(get_pinned_commit fbgemm)
+  pip_uninstall torchrec-nightly
+  pip_uninstall fbgemm-gpu-nightly
+  function doit() {
+    pip_install --no-use-pep517 --user "git+https://github.com/pytorch/FBGEMM.git@${commit}#egg=fbgemm-gpu&subdirectory=fbgemm_gpu"
+    pip_install --no-use-pep517 --user "git+https://github.com/pytorch/torchrec.git@${commit}"
+  }
+  if [[ "$1" == "cuda" ]]; then
+    # TODO: This is better to be passed as a parameter from _linux-test workflow
+    # so that it can be consistent with what is set in build
+    TORCH_CUDA_ARCH_LIST="8.0;8.6" doit
```

Comment:
Hmm, there is already `torch.cuda.get_gencode_flags()`, but worst case you can do something like
```suggestion
    TORCH_CUDA_ARCH_LIST=$(python3 -c "import torch;props=torch.cuda.get_device_properties(0);print(f'{props.major}.{props.minor}')") doit
```

Reply:
I cargo culted this from the torchaudio instructions. Let me try your suggestion.

Reply:
Hmm, I don't think your suggestion is right. What if we're doing a build from a CPU runner? You won't specify arch list in that case.

Reply:
I just deleted this entirely and pray